### PR TITLE
Create tests for the DateUtils class

### LINKED ISSUE #787: Add unit tests for DateUtils class
### Describe the chore
Add unit tests for DateUtils class: https://github.com/project-travel-mate/Travel-Mate/blob/master/Android/app/src/main/java/utils/DateUtils.java

Code coverage should be ~90%

### PR DESCRIPTION
**Pull request by fokion**
## Description

Create tests for the DateUtils class

Fixes #787 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Unit tests.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`
- [x] `./gradlew test`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by fokion** [comment]
but this is a unit test so it has to be on the test directory according to the documentation https://developer.android.com/training/testing/unit-testing/local-unit-tests.html . The test is running if you run gradlew test

File: Android/app/src/test/java/utils/DateUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+public class DateUtilsTest {
+
+    @Test
+    public void rfc3339ToMills() {
+        assertEquals(1572374198000L , DateUtils.rfc3339ToMills("2019-10-29T18:36:38.000GMT").longValue());
+    }
+    @Test
+    public void getDate_with_mils() {
+        assertEquals("29 Oct 2019" , DateUtils.getDate(1572374198000L));
```
Comment (Swati4star) [inline Android/app/src/test/java/utils/DateUtilsTest.java]: Can you also add some negative test cases for invalid dates

Comment (Swati4star) [inline Android/app/src/test/java/utils/DateUtilsTest.java]: Can you also add some negative test cases for invalid dates

**Comment by Swati4star** [comment]
Can you also add `./gradlew test` to travis.yml (https://github.com/project-travel-mate/Travel-Mate/blob/master/.travis.yml) at end so that the tests are executed by travis also.

File: Android/app/src/test/java/utils/DateUtilsTest.java (hunk 1 above)
Comment (Swati4star) [reply Android/app/src/test/java/utils/DateUtilsTest.java]: This isnt resolved yet?

File: Android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```
Comment (Swati4star) [inline Android/gradle/wrapper/gradle-wrapper.properties]: is this change needed?

File: Android/app/src/test/java/utils/DateUtilsTest.java (hunk 1 above)
Comment (Swati4star) [reply Android/app/src/test/java/utils/DateUtilsTest.java]: How is it resolved now?

Comment (Swati4star) [reply Android/app/src/test/java/utils/DateUtilsTest.java]: @fokion Why are you marking this as resolved?

**Comment by fokion** [comment]
I have finished with the requested changes

**Comment by Swati4star** [comment]
No @fokion, you havent.

**Comment by fokion** [comment]
Can you clarify what do you need ? I have updated the commit to remove that entry for gradle and the command is on the travis configuration

File: Android/app/src/test/java/utils/DateUtilsTest.java (hunk 1 above)
Comment (fokion) [reply Android/app/src/test/java/utils/DateUtilsTest.java]: https://github.com/project-travel-mate/Travel-Mate/commit/1b86d19bcb956af2745d37bfb6f4d117b3c2512c

Comment (fokion) [reply Android/app/src/test/java/utils/DateUtilsTest.java]: that is the reason why I resolve it. there are two sets of invalid tests that cover a wrong date format and a text that is invalid as it is not a date at all.
